src/test/java/org/javarosa/core/test/Scenario.java

```diff
@@ -438,6 +439,14 @@ public static Scenario init(String formName, XFormsElement form) throws IOExcept
         return Scenario.init(formFile);
     }
 
+    public static FormDef createFormDef(String formName, XFormsElement form) throws IOException, XFormParser.ParseException {
+        Path formFile = createTempDirectory("javarosa").resolve(formName + ".xml");
+        String xml = form.asXml();
+        System.out.println(xml);
```

Comment:
We should probably clean these up at some point (it's also in the source `init` implementation)